packages/azure-services/src/storage/cosmos-container-client.spec.ts

```diff
@@ -16,12 +17,17 @@ type OperationCallback = (...args: any[]) => Promise<CosmosOperationResponse<any
 const dbName = 'dbName';
 const collectionName = 'collectionName';
 const partitionKey = 'default-partitionKey';
+const startTime = new Date();
```

Comment:
use a constant time. startTime value should not change based on when you are running the test.